src/main/java/com/hazelcast/kubernetes/KubernetesProperties.java

```diff
@@ -104,6 +104,12 @@
      */
     public static final PropertyDefinition KUBERNETES_API_TOKEN = property("api-token", STRING);
 
+    /**
+     * <p>Configuration key: <tt>hazelcast-service-port</tt></p>
+     * If specified, its value defines the endpoint port of the service (overriding the default).
```

Comment:
"If specified with a value > 0, ..."
or similar wording, just to clarify that with zero you get the default.

Reply:
fixed